.github/workflows/build_and_test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
   build_and_test:
     timeout-minutes: 100
-    runs-on: ubuntu-20.04
+    runs-on: gasp
```

Comment:
Probably a question for @mgaravin -- how would this work for MEV/ES2K target? Would CI builds still be supported through the Github runners?

Reply:
No clue. We'll have to wait for the CI/CD boffins to weigh in.

Reply:
Yes, we have Intel shared GitHub runners provided by 1source team! I'm trying to make that work, if not i have to add dedicated "self-hosted" runner and execute the github actions.

Pls. note: Self-host runner can't run all opensource github actions ( as few actions as expected to run only in opensource environment)! have to check and do modifications in innersource based on our need!

Reply:
And, probably, we should not upstream innersource github actions to opensource, as it was customized to run only on innersource environment

Reply:
I have backed out the changes to the github actions